### PR TITLE
Implement TreeNodeObject -> ObjectCtrlInfo extension method

### PR DESCRIPTION
**Pull request by starstormhun**

### CODE REVIEW

File: src/Shared.Core/KoikatuAPIBase.cs [hunk 1]
```diff
@@ -33,7 +33,7 @@ public partial class KoikatuAPI
         /// THIS VALUE WILL NOT BE READ FROM THE INSTALLED VERSION, YOU WILL READ THE VALUE FROM THIS VERSION THAT YOU COMPILE YOUR PLUGIN AGAINST!
         /// More info: https://stackoverflow.com/questions/55984/what-is-the-difference-between-const-and-readonly
         /// </summary>
-        public const string VersionConst = "1.36";
+        public const string VersionConst = "1.37";
```
Comment (ManlyMarco) [inline src/Shared.Core/KoikatuAPIBase.cs:36]:
```suggestion
        public const string VersionConst = "1.36";
```

**Review by ManlyMarco — APPROVED** [review]

**Comment by takahiro0327** [comment]
Sorry, I'm really just curious, so I'll ask.
Why don't you use Studio.instance.dicInfo?
Is there a limit to the games, timing or performance you can use it for?

Is there something wrong with the following code?
```
public static ObjectCtrlInfo GetOCIFromNode( TreeNodeObject obj )
{
    ObjectCtrlInfo oci = null;
    Studio.Studio.instance?.dicInfo?.TryGetValue(obj, out oci);
    return oci;
}
```

image

**Comment by starstormhun** [comment]
Well, I guess because neither of us knew it existed
I had scanned that class so many times but it's so obscure

**Comment by starstormhun** [comment]
Yep just tried, it works, thanks @takahiro0327

**Comment by takahiro0327** [comment]
Thanks. That clears things up.

**Pull request merged**